sleap/gui/learning/dialog.py

```diff
@@ -717,6 +725,20 @@ def run(self):
             win.setWindowTitle("Inference Results")
             win.exec_()
 
+    def copy(self):
+        """Copy scripts and configs to clipboard"""
+        pipeline_form_data = self.pipeline_form_widget.get_form_data()
+        items_for_inference = self.get_items_for_inference(pipeline_form_data)
+        config_info_list = self.get_every_head_config_data(pipeline_form_data)
+        output_json = pipeline_form_data
+        for config_info in config_info_list:
+            config_info = config_info.config.to_json()
+            dict_config = json.loads(config_info)
+            output_json.update(dict_config)
```

Comment:
The `config_info_list` should contain the config info for every tab that is shown:
https://github.com/talmolab/sleap/blob/1151a95cd699904d5ba28b591bcb21c2faf2ca89/sleap/gui/learning/dialog.py#L480-L488

For top-down models, there are two tabs shown (one for the centroid and one for the centered instance). These tabs have the exact same keys when converted to dictionaries, thus the `update` method overwrites the keys with only the last config info... Notice that when we copy/past for a top-down model, we only ever receive the info for the centered-instance model.

Reply:
I've fixed it! Do I need to copy
```python
items_for_inference = self.get_items_for_inference(pipeline_form_data)
``` 
to clipboard as well? Since when I print it out this items_for_inference is empty.

Reply:
I think this is plenty of info!